testsuite/libmapi/mapi_idset.c

```diff
@@ -60,6 +60,67 @@ START_TEST (test_IDSET_parse) {
 
 } END_TEST
 
+START_TEST (test_IDSET_remove_rawidset) {
+	const uint64_t		ids[] = {0x1d0401000000,
+					0x1e0401000000,
+					0x1f0401000000,
+					0x200401000000,
+					0x210401000000,
+					0x220401000000,
+					0x230401000000,
+					0x240401000000};
+	int			i;
+	size_t			ids_size = sizeof(ids)/sizeof(uint64_t);
+	struct idset		*idset_in;
+	uint16_t		repl_id = 0x0001;
+	struct globset_range	*range;
+	struct rawidset		*rawidset_in, *rawidset_rm_0, *rawidset_rm_1, *rawidset_rm_2;
+
+	/* Generate idsets */
+	rawidset_in = RAWIDSET_make(mem_ctx, true, false);
```

Comment:
It'd be great to assert not null here to avoid aborting in out of memory cases. Not mandatory though. (It applies to all)